samples/sqltransport-nhpersistence-outbox-ef/Version_2.1/Receiver/OrderSubmittedHandler.cs

```diff
@@ -1,12 +1,15 @@
 using System;
 using NServiceBus;
-using System.Data.Common;
-using NServiceBus.Persistence.NHibernate;
 
 public class OrderSubmittedHandler : IHandleMessages<OrderSubmitted>
 {
     public IBus Bus { get; set; }
-    public NHibernateStorageContext StorageContext { get; set; }
+    private readonly ReceiverDataContext ctx;
+
+    public OrderSubmittedHandler(ReceiverDataContext ctx)
```

Comment:
Mixing constructor and property injection will break some containers. U need to use one of the other